src/components/EntityInfo/EntityMenu.vue

```diff
@@ -41,6 +41,48 @@
       </v-tooltip>
     </span>
 
+    <!-- Amalgamate -->
+    <span v-if="isBusiness && !isHistorical">
+      <template v-if="!isAdminFrozen && !isPendingDissolution">
+        <v-btn
+          id="amalgamate-button"
+          small
+          text
+          color="primary"
+          :disabled="!isAllowed(AllowableActions.AMALGAMATION)"
+          @click="goToAmalgamationSelection()"
+        >
+          <v-icon medium>mdi-domain-plus</v-icon>
+          <span class="font-13 ml-1">Amalgamate</span>
+        </v-btn>
+      </template>
+      <template v-else>
```

Comment:
There's a better way to make a tooltip conditional -- just disable it.

Reply:
Example: https://github.com/bcgov/business-create-ui/blob/0902aa5a7e631ab0abc81567681a77364da6741f/src/components/Amalgamation/BusinessStatus.vue#L8

Reply:
This is awesome btw Sev. Was wondering if there was an easy way, I didn't know about this. Thanks!

Reply:
Fixed, thanks.